swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
+from swap_meet.item import Item 
```

Comment:
so here, we don't actually need to import the literal class Item itself. We aren't instantiating an instance of Item (ie, Item(category="antique",3) ) inside the Vendor anywhere.

There are "items" coming in through parameters, but the class "blueprint" itself can't tell that. On line 7 ` def add(self, item)` the `item` parameter is simply a placeholder. It could be called `banana`, `apple`, and when this method is actually called we could pass an integer, a string, a dictionary, etc. basically, the method itself doesn't know what these parameter values will be. 

So unless we are actually creating an instance _inside_ vendor (like our composition examples), we don't need to import it